plugin/PDFDocumentToCPage.qml

```diff
@@ -35,6 +35,13 @@ Page {
         //% "Index"
         header: PageHeader { title: qsTrId( "sailfish-office-he-pdf_index" ); }
 
+        ViewPlaceholder {
+            id: placeholder
+            //% "Document has no table of content"
+            text: qsTrId("sailfish-office-me-no-toc")
+        }
+        onModelChanged: placeholder.enabled = !model || (model.count == 0)
```

Comment:
This is strange, I agree. I put in the PR comment that the enabled attribute of the placeholder is not set by binding since the tocmodel object comes from a different thread and QML complains about signaling from a different thread. Thus, enabled attribute is set by javascript by reading the count value when model is set instead.

Maybe a comment in the QML code would be great to avoid later wondering about this. Or would you prefer to change the C++ part to avoid having the tocmodel object living in the other thread as it is now ? I choose the onModelChanged solution because it doesn't change big parts of the C++ and do the job without any warning in one line, which seems fine to me (but unusual in the way to write it).